server/socketManager.js

```diff
@@ -0,0 +1,60 @@
+const { Server } = require('bweb');
+const assert = require('bsert');
+
+class SocketManager extends Server {
+  /**
+   * Create an http server.
+   * @constructor
+   * @param {Object} options
+   */
+  constructor(options) {
+    // super(new SocketManagerOptions(options));
```

Comment:
Dead code

Reply:
This file is just scaffolding. Still WIP 😄

Reply:
👍